product_sale_description/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Product Sale Description",
     "summary": "Long and short description for products",
-    "version": "16.0.1.0.1",
+    "version": "17.0.1.0.1",
```

Comment:
@Deriman-Alonso Careful. This change should go into migration commit instead of pre-commit one.

Moreover, you should reset version to '17.0.1.0.0`

Reply:
@rousseldenis Hi, thats changed, thanks.